source/_components/plum_lightpad.markdown

```diff
@@ -0,0 +1,54 @@
+---
+layout: page
+title: "Plum Lightpad"
+description: "Instructions on setting up Plum Lightpads within Home Assistant."
+date: 2018-10-15 17:58
+sidebar: true
+comments: false
+sharing: true
+footer: true
+ha_category: Hub
+ha_iot_class: "Local Push"
+featured: true
+ha_release: "0.80"
```

Comment:
0.80 is the current release, not the next one.

Reply:
I wasn't sure, what should I put here?  It's an easy update for whenever it goes in

Reply:
For now it's ok, just a reminder to update when it's done.